package.json

```diff
@@ -9,6 +9,8 @@
         "@babel/polyfill": "^7.4.0",
         "@babel/preset-env": "^7.4.2",
         "@babel/register": "^7.4.0",
+        "@pattern-lab/patternengine-node-mustache": "^2.0.0-alpha.2",
+        "@pattern-lab/patternlab-node": "3.0.0-alpha.8",
```

Comment:
make sure to remove "patternlab-node": "^2.12.0",

Reply:
Removed in recent commit